packages/utils/src/utils/split-utxos.ts

```diff
@@ -0,0 +1,36 @@
+import { ErrorCode, FuelError } from '@fuel-ts/errors';
+import type { AbstractAddress } from '@fuel-ts/interfaces';
+import type { BN } from '@fuel-ts/math';
+
+/**
+ * This function allows a consumer to split a UTXO into multiple smaller UTXOs
```

Comment:
UTXOs only become such after a transaction has been processed. This utility produces transaction outputs that will become UTXOs if the user puts them in a transaction they submit.

>  consumer to split a UTXO

With regards to this, I wrote a comment [here](https://github.com/FuelLabs/fuels-ts/pull/3435/files#r1871389903).

```suggestion
 * This function allows a consumer to split a UTXO into multiple smaller transaction outputs.
```

Reply:
This is no longer relevant 7165a1a85